### DISCUSSION TITLE
PDF Loader Fails to Load my PDF without an Error

### DISCUSSION THREAD

**Discussion by lanceg321** [FlowiseAI/Flowise#618] [Q&A]
I am new to Flowise, so please forgive me there is an obvious answer but I searched online and forums and was unable to find the solution.  I created a based QA conversational bot that I want to load multiple PDF's into to query for answers.  Strangley, some PDF's load just fine and other fail to load but with no error.  When I select the file via PDF loader widget, and select okay, the folder window closes but the widget still shows a red asterisk by the pdf file option in the widget showing that it was ignored.  I tested for size limits and I have another PDF that is 3x as large and loads fine but the two that I need to load do not work.  I'm attaching a copy of the one that works and the one that doesn't for reference.  Any guidance on how I can upload the one that doesn't work would be super helpful.  I appreciate it!
Lance[](url)

[Not Loading_50051-E,_Series_5_Dental_Chair_Installation_Instructions.pdf](https://github.com/FlowiseAI/Flowise/files/12164322/Not.Loading_50051-E._Series_5_Dental_Chair_Installation_Instructions.pdf)
[Loads fine_DCI_Reliance_Order_Guide_1.pdf](https://github.com/FlowiseAI/Flowise/files/12164323/Loads.fine_DCI_Reliance_Order_Guide_1.pdf)

**Comment by HenryHengZJ** [comment]
have you tried to turn of use legacy options?
<img width="692" alt="image" src="https://github.com/FlowiseAI/Flowise/assets/26460777/f5764304-7d54-4682-8a60-4134787835cc">

**Reply by lanceg321** [reply]
Thank you for the response.  Yes, I tried both modes and still not able to upload.  No error reported, just drops back to the PDF loader widget and shows no file uploaded.  I appreciate it.  If you have any other ideas, I would love to hear them.
